scripts/install-envoy-validate-plugin.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -euo pipefail
+
+source scripts/executor-info.sh
+
+PLUGIN_VERSION=$1
+
+if ! type "protoc-gen-validate" > /dev/null 2>&1; then
```

Comment:
Maybe we should remove the check at all. `go install` will not rebuild anyway if it is already there, and with the check there might be problems where the wrong version is already installed.
But can you add a check whether `go` itself is installed, and error with a good message if it is not?

Reply:
Yeah, that is a good idea. I can also check to ensure the correct version of Go is installed as well